inscriptions/ISic003429.xml

```diff
@@ -79,12 +79,8 @@
                         <objectDesc>
                             <supportDesc>
                                 <support>
-                                    <material key="limestone" ref="http://www.eagle-network.eu/voc/material/lod/66.html">
-                                        limestone
-                                    </material>
-                                    <objectType key="sarcophagus" ref="http://www.eagle-network.eu/voc/objtyp/lod/214.html">sarcophagus;
-                                        tabula ansata
-                                    </objectType>
+                                    <material key="limestone" ref="http://www.eagle-network.eu/voc/material/lod/66.html">limestone</material>
+                                    <objectType key="sarcophagus" ref="http://www.eagle-network.eu/voc/objtyp/lod/214.html">sarcophagus; tabula ansata</objectType>
```

Comment:
delete the 'tabula ansata' bit from the object type,a nd add a note in the `<layout>` in `<p>` that the text is set out within a tabula ansata